libraries/exporters/src/main/java/com/siemens/sw360/exporter/ProjectExporter.java

```
@@ -91,53 +113,92 @@ public int getColumns() {
+                            )));
+                            break;
+                        case ATTACHMENTS:
+                            row.add(project.attachments.size()+"");
+                            break;
+                        default:
+                            row.add(fieldValueAsString(fieldValue));
+                    }
+                } else {
                     row.add("");
                 }
-
             }
 
             return row;
         }
 
-        private List<String> getReleases(Set<String> ids) {
-            if (ids == null) return Collections.emptyList();
-
+        private SubTable makeRowForProjectOnly(Project project){
+            releases = getReleases(project);
+            return  new SubTable(makeRowForProject(project));
+        }
 
+        private List<Release> getReleases(Project project) {
             List<Release> releasesByIdsForExport;
             try {
-                releasesByIdsForExport = client.getReleasesByIdsForExport(ids);
+                releasesByIdsForExport = componentClient.getReleasesByIdsForExport(nullToEmptySet(project.releaseIds));
             } catch (TException e) {
                 log.error("Error fetching release information", e);
-                releasesByIdsForExport=Collections.emptyList();
+                releasesByIdsForExport = Collections.emptyList();
             }
+            return releasesByIdsForExport;
+        }
 
-            List<String> releaseNames = new ArrayList<>(ids.size());
-
-            for (Release release : releasesByIdsForExport) {
-                releaseNames.add(printName(release));
+        private List<Project> getProjects(Set<String> ids, User user){
+            List<Project> projects;
+            try {
+                projects = projectClient.getProjectsById(ids, user);
+            } catch (TException e) {
+                log.error("Error fetching linked projects.", e);
```

Same as above: this should fail hard